src/io/flutter/view/InspectorPanel.java

```diff
@@ -495,6 +496,7 @@ void registerShortcuts() {
       DebuggerUIUtil.registerActionOnComponent(InspectorActions.JUMP_TO_TYPE_SOURCE, this, this);
     }
 
+    @SuppressWarnings("EmptyMethod")
```

Comment:
noob question. this method doesn't seem empty?

Reply:
Right.  IDEA flags it though.  This gets the inspection to shut up.